gnd/src/main/java/com/google/android/gnd/GndApplication.java

```diff
@@ -16,30 +16,31 @@
 
 package com.google.android.gnd;
 
+import android.app.Application;
 import android.content.Context;
 import android.os.StrictMode;
 import android.util.Log;
 import androidx.annotation.NonNull;
+import androidx.hilt.work.HiltWorkerFactory;
 import androidx.multidex.MultiDex;
 import androidx.work.Configuration;
-import androidx.work.WorkManager;
 import com.akaita.java.rxjava2debug.RxJava2Debug;
 import com.facebook.stetho.Stetho;
-import com.google.android.gnd.inject.GndWorkerFactory;
 import com.google.android.gnd.rx.RxDebug;
 import com.google.firebase.crashlytics.FirebaseCrashlytics;
-import dagger.android.AndroidInjector;
-import dagger.android.support.DaggerApplication;
+import dagger.hilt.android.HiltAndroidApp;
 import io.reactivex.plugins.RxJavaPlugins;
 import javax.inject.Inject;
+import org.jetbrains.annotations.NotNull;
 import timber.log.Timber;
 
 // TODO: When implementing background data sync service, we'll need to inject a Service here; we
 // should then extend DaggerApplication instead. If MultiDex is still needed, we can install it
 // without extending MultiDexApplication.
```

Comment:
We should update this comment as it is out-of-date now

Reply:
@gino-m Is this comment (or something similar) still required?

Reply:
I think it can be removed, since we're not using Services, and Workers appear to be injected here w/o additional changes. Thanks!